FamilySearchIosSampleApp/TreeTVC.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  TreeVC.swift
+//  FamilySearchIosSampleApp
+//
+//  Created by Eduardo Flores on 6/6/16.
+//  Copyright © 2016 FamilySearch. All rights reserved.
+//
+
+import UIKit
+
+class TreeTVC: UITableViewController {
+    
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    
+    var user : User!
+    
+    var personArray = NSArray()
+    
+    var accessToken : String!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // get the access token from NSUserDefaults
+        let preferences = NSUserDefaults.standardUserDefaults()
+        accessToken = preferences.stringForKey(Utilities.KEY_ACCESS_TOKEN)
+        
+        // get url for family tree from Collections
+        Utilities.getUrlsFromCollections({ (collectionsResponse, error) -> Void in
```

Comment:
That makes more sense as to why it was done the way it was done.  The downside to this approach though is the UI code looks a lot more complicated than it needs to because of all of the closures, and I don't have a single place to go look at how I load my urls.  As a developer, I'd rather go to a single place to see how to fetch urls and then I can dig into individual handling of those urls if I want but I'd rather understand the url fetching once rather than pay the price over and over with each ui piece.  That's my thoughts anyway, @misbach might have different feelings.